_layouts/main.html

```diff
@@ -1,26 +1,33 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>{{ page.title }}</title>
-    <link rel="stylesheet" href="/assets/css/styles.css">
-      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
-    <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
-    <!--{% feed_meta %}-->
-    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
-    <script src="/assets/js/navbar-burger.js"></script>
-    {% seo %}
-  </head>
-  <body>
-    <section class="hero is-fullheight">    
-      {% include navigation.html icon-size="lg" %}
-      <div class="hero-body">
-        <div class="container">
-          {{ content }}
-        </div>
-      </div>
-      {% include footer.html icon-size="lg" %}
-    </section>
-  </body>
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<title>{{ page.title }}</title>
+		<link rel="stylesheet" href="/assets/css/styles.css" />
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css"
+		/>
+		<!--{% feed_meta %}-->
+		<script
+			defer
+			src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"
+		></script>
+		<script src="/assets/js/navbar-burger.js"></script>
+		{% seo %}
+	</head>
+	<body>
+		<section class="hero is-fullheight">
+			{% include navigation.html icon-size="lg" %}
+			<div class="hero-body">
+				<div class="container">{{ content }}</div>
+			</div>
+			{% include footer.html icon-size="lg" %}
+		</section>
+	</body>
```

Comment:
I can't easily see the actual changes because of the new indent. Would it be possible to revert the indent to 2 spaces?

Reply:
No changes were made. The file was automatically formatted when I opened it. I will format it back to 2 spaces.